src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -51,7 +51,11 @@ public function __construct(ContainerInterface $container, $resource, array $opt
     public function getRouteCollection()
     {
         if (null === $this->collection) {
+            $this->collection = new RouteCollection();
```

Comment:
It is possible, but loading occurs only once. This does not change. What changes is that if the loader fails, we do not retry a second time (which never worked and can't work)